docs/Makefile

```diff
@@ -1,13 +1,7 @@
 # You can set these variables from the command line.
-ifeq ($(OS),Windows_NT)
```

Comment:
This is useful on Windows, lets leave it as is

Reply:
@zimnx I bet none of use has ever tried it and it came from upstream, I'd trust David on that

Reply:
I'm ok with leaving it for this project, but we are recommending the following now since 1.6:

Changelog:

<img width="732" alt="image" src="https://github.com/scylladb/scylla-operator/assets/9107969/fb27f33d-b902-4297-9007-79c1fcce6a88">

Migration guide:

<img width="748" alt="image" src="https://github.com/scylladb/scylla-operator/assets/9107969/f08e3759-5ea3-4f03-9f6c-4b28b5507040">

Reply:
Ok, I recall it was added in 1.5 explicltly https://github.com/scylladb/scylla-operator/pull/1308#issuecomment-1669262385, if recommendation changed then I'm fine with removing it.